src/view/xwayland.cpp

```diff
@@ -356,7 +359,7 @@ class wayfire_xwayland_view : public wayfire_xwayland_view_base
         {
             last_server_width = xw->size_hints->base_width;
             last_server_height = xw->size_hints->base_height;
-            send_configure();
+            send_configure(XCB_CONFIG_WINDOW_X | XCB_CONFIG_WINDOW_Y);
```

Comment:
huh, i think here we want to change width and height as well?

Reply:
Yes, it seems so.